cypress/e2e/example.cy.ts

```diff
@@ -3,6 +3,6 @@
 describe('My First Test', () => {
   it('visits the app root url', () => {
     cy.visit('/');
-    cy.contains('h3', 'Documentation');
+    cy.contains('.match', 'Counter-Strike');
```

Comment:
A better test here would be to just check if the element `.match` exists at all since this test would fail if the API returned a different game, for example. You could do `cy.get('.match')`. Even better would be to add an attribute like `data-cy="match"` and test that exists as the class could be used for CSS and might change, therefore breaking the test.